.devcontainer/devcontainer.json

```diff
@@ -62,14 +62,10 @@
 				"njpwerner.autodocstring",
 				"VisualStudioExptTeam.vscodeintellicode",
 				"yzhang.markdown-all-in-one",
```

Comment:
do we need this?

Reply:
Not really, I removed on extension (there were two) but the OOTB VS Code markdown support is good enough in my opinion.